cards/src/main/resources/cards/custom/group0/other_death_knight/basic/minion_archerus_deathcharger.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Archerus Deathcharger",
+  "baseManaCost": 3,
+  "type": "MINION",
+  "heroClass": "SPIRIT",
+  "baseAttack": 3,
+  "baseHp": 4,
+  "rarity": "FREE",
+  "description": "Charge. Can't attack heroes.",
```

Comment:
I was thinking that too, but it isn't exactly the same because this also can't attack heroes on subsequent turns. It would be a buff just giving it Rush, and also just a straight up more powerful card than Rabid Worgen for warrior.